### PR TITLE
BUMP 4.4.0b0

### PR DESCRIPTION
**Pull request by juliusgeo**

### CODE REVIEW

File: doc/changelog.rst [hunk 1]
```diff
@@ -6,6 +6,11 @@ Changes in Version 4.4
 
 - Added support for passing a list containing (key, direction) pairs
   or keys to :meth:`~pymongo.collection.Collection.create_index`.
+- **BETA** Added support for range queries on client side field level encrypted collections.
+- pymongocrypt 1.5.0 or later is now required for client side field level
+  encryption support.
+- Improved support for Pyright to improve type support for IDEs like Intellisense.
```
Comment (ShaneHarvey) [inline doc/changelog.rst:12]: Is Intellisense an IDE? Maybe "IDEs like Visual Studio and VS Code."

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]

File: doc/changelog.rst (hunk 1 above)
Comment (juliusgeo) [reply doc/changelog.rst:12]: You're right, I got it confused. Fixed.

File: doc/changelog.rst [hunk 2]
```diff
@@ -6,6 +6,12 @@ Changes in Version 4.4
 
 - Added support for passing a list containing (key, direction) pairs
   or keys to :meth:`~pymongo.collection.Collection.create_index`.
+- **BETA** Added support for range queries on client side field level encrypted collections.
+- pymongocrypt 1.5.0 or later is now required for client side field level
+  encryption support.
+- Improved support for Pyright to improve typing support for IDEs like Visual Studio Code or Visual Studio.
+- Added support for type-checking with MyPy "strict" mode (`--strict`).
```
Comment (ShaneHarvey) [inline doc/changelog.rst:13]: I'd say "Improved support" here. From Steve's comments it sounds like we need to do work work and testing to resolve other issues besides the CodecOptions piece.

Comment (juliusgeo) [reply doc/changelog.rst]: Fixed.

**Review by ShaneHarvey — APPROVED** [review]

**Pull request merged**